### PR TITLE
Polish docs for the v1.1.0 release

### PR DESCRIPTION
**Pull request by guangbochen**
**Description:**
- update image asserts with v1.1.0-related screenshot.
- fixing formatting issues
- added Rancher v2.6.9 & Harvester v1.1.0 support matrix

### CODE REVIEW

File: docs/advanced/pcidevices.md [hunk 1]
```diff
@@ -1,48 +1,55 @@
...
+_Available as of v1.1.0_
+
+A `PCIDevice` in Harvester represents a host device with a PCI address. 
+The devices can be passed through the hypervisor to a VM by creating a `PCIDeviceClaim` resource, 
 or by using the UI to enable passthrough. Passing a device through the hypervisor means that 
 the VM can directly access the device, and effectively owns the device. A VM can even install 
-it's own drivers for that device.
+its own drivers for that device.
+
+This is accomplished using the `pcidevices-controller` addon.
```
Comment (vickyhella) [inline docs/advanced/pcidevices.md:17]:
```suggestion
This is accomplished by using the `pcidevices-controller` addon.
```

File: docs/advanced/pcidevices.md [hunk 2]
```diff
@@ -1,48 +1,55 @@
...
+To use the PCI devices feature, users need to enable the `pcidevices-controller` addon first.
 
-Go to the Advanced -> PCI Devices UI:
+![](/img/v1.1/vm-import-controller/EnableAddon.png)
 
-![](/img/v1.1/pcidevices/advanced-pcidevices-index.png)
+## Enabling Passthrough on a PCI Device
 
-Search for your device by vendor name (e.g. NVIDIA, Intel, etc.) or device name.
+1. Now go to the `Advanced -> PCI Devices` UI:
```
Comment (vickyhella) [inline docs/advanced/pcidevices.md:25]:
```suggestion
1. Now go to the `Advanced -> PCI Devices` page:
```

File: docs/advanced/pcidevices.md [hunk 3]
```diff
@@ -1,48 +1,55 @@
...
 
-Then click "Enable Passthrough", read the warning*, and if you still want to enable for those devices, click "Enable" and wait until all the devices say "Enabled"
+1. Select the devices you want to enable for passthrough:
 
-![](/img/v1.1/pcidevices/enable-pcidevices-inprogress.png)
+   ![](/img/v1.1/pcidevices/select-pcidevices.png)
 
-*The warning looks like this:
-![](/img/v1.1/pcidevices/warning.png)
+1. Then click **Enable Passthrough**, read the warning message, if you still want to enable these devices, click **Enable** and wait for all devices to be `Enabled`.
```
Comment (vickyhella) [inline docs/advanced/pcidevices.md:37]:
```suggestion
1. Then click **Enable Passthrough** and read the warning message. If you still want to enable these devices, click **Enable** and wait for all devices to be `Enabled`.
```

File: docs/advanced/pcidevices.md [hunk 4]
```diff
@@ -1,48 +1,55 @@
...
+   :::caution
+   Please be careful not to use `host-owned` PCI devices(e.g., management and VLAN NICs). Incorrect device allocation may cause damage to your cluster, including node failure.
```
Comment (vickyhella) [inline docs/advanced/pcidevices.md:39]:
```suggestion
   Please do not use `host-owned` PCI devices (e.g., management and VLAN NICs). Incorrect device allocation may cause damage to your cluster, including node failure.
```

File: docs/advanced/pcidevices.md [hunk 5]
```diff
@@ -1,48 +1,55 @@
...
 
+   ![](/img/v1.1/pcidevices/enable-pcidevices-inprogress.png)
 
-![](/img/v1.1/pcidevices/enable-pcidevices-done.png)
+   ![](/img/v1.1/pcidevices/enable-pcidevices-done.png)
 
 ## Attaching PCI Devices to a VM
 
-Go to the VM UI and select "Edit Config"
+After enabling these PCI devices, you can click on the VM page and select **Edit Config** to pass these devices.
```
Comment (vickyhella) [inline docs/advanced/pcidevices.md:48]:
```suggestion
After enabling these PCI devices, you can navigate to the **Virtual Machines** page and select **Edit Config** to pass these devices.
```

File: docs/index.md [hunk 6]
```diff
@@ -75,22 +75,24 @@ During the installation, you can either choose to form a new cluster or join the
...
-13. Confirm the installation options and Harvester will be installed to your host. The installation may take a few minutes to complete.
-14. Once the installation is complete, the host will restart, and a console UI with management URL and status will be displayed. <small>(You can Use F12 to switch between the Harvester console and the Shell).</small>
-15. The default URL of the web interface is `https://your-virtual-ip`.
+5. Recommended choosing a separate disk for storing VM data.
+   ![iso-install-disk.png](/img/v1.1/install/iso-select-data-disk.png )
+6. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster, in case the DHCP is used, the user must make sure the DHCP server always offers the same IP for the same Node. Due to a changed Node IP the related Node can not join the cluster, or even break the cluster)</small>.
```
Comment (vickyhella) [inline docs/index.md:80]:
```suggestion
6. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster. If DHCP is used, users must make sure the DHCP server always offers the same IP for the same Node. If Node IP is changed, the related Node cannot join the cluster and might even break the cluster)</small>.
```

File: docs/index.md [hunk 7]
```diff
@@ -75,22 +75,24 @@ During the installation, you can either choose to form a new cluster or join the
...
+9. Configure the `cluster token`. This token will be used for adding other nodes to the cluster.
+10. Configure the login password of the host. The default SSH user is `rancher`.
+11. Recommended configuring the NTP server to make sure all nodes' times are synchronized. This defaults to `0.suse.pool.ntp.org`.
+12. (Optional) If you need to use an HTTP proxy to access the outside world, enter the proxy URL address here. Otherwise, leave this blank.
+13. (Optional) You can choose to import SSH keys from a remote server URL. Your GitHub public keys can be used with `https://github.com/<username>.keys`.
+14. (Optional) If you need to customize the host with a [Harvester configuration](./install/harvester-configuration.md) file, enter the HTTP URL here.
+15. Confirm the installation options and Harvester will be installed to your host. The installation may take a few minutes to complete.
```
Comment (vickyhella) [inline docs/index.md:91]:
```suggestion
15. Confirm the installation options and Harvester will be installed on your host. The installation may take a few minutes to complete.
```

File: docs/install/iso-install.md [hunk 8]
```diff
@@ -31,9 +31,11 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
 1. Choose the installation device on which the Harvester cluster will be installed
       - Note: By default, Harvester uses [GPT](https://en.wikipedia.org/wiki/GUID_Partition_Table) partitioning schema for both UEFI and BIOS. If you use the BIOS boot, then you will have the option to select [MBR](https://en.wikipedia.org/wiki/Master_boot_record).
    ![iso-install-disk.png](/img/v1.1/install/iso-install-disk.png)
-1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `harvester-mgmt`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster, in case the DHCP is used, the user must make sure the DHCP server always offers the same IP for the same Node. Due to a changed Node IP the related Node can not join the cluster, or even break the cluster)</small>.
+1. Recommended choosing a separate disk for storing VM data.
```
Comment (vickyhella) [inline docs/install/iso-install.md:34]:
```suggestion
1. Recommended choosing a separate disk to store VM data.
```

File: docs/install/iso-install.md [hunk 9]
```diff
@@ -31,9 +31,11 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+   ![iso-install-disk.png](/img/v1.1/install/iso-select-data-disk.png )
+1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster, in case the DHCP is used, the user must make sure the DHCP server always offers the same IP for the same Node. Due to a changed Node IP the related Node can not join the cluster, or even break the cluster)</small>.
```
Comment (vickyhella) [inline docs/install/iso-install.md:36]:
```suggestion
1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster. If DHCP is used, users must make sure the DHCP server always offers the same IP for the same Node. If Node IP is changed, the related Node cannot join the cluster and might even break the cluster)</small>.
```

File: docs/install/iso-install.md [hunk 10]
```diff
@@ -48,3 +50,8 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
    ![iso-installed.png](/img/v1.1/install/iso-installed.png)
 1. You will be prompted to set the password for the default `admin` user when logging in for the first time.
    ![first-login.png](/img/v1.1/install/first-time-login.png)
+
+
+:::note
+In some cases, if you are using an older VGA connector, you may encounter an `panic: invalid dimensions` error with ISO installation, see issue [#2937](https://github.com/harvester/harvester/issues/2937#issuecomment-1278545927) for a workaround.
```
Comment (vickyhella) [inline docs/install/iso-install.md:56]:
```suggestion
In some cases, if you are using an older VGA connector, you may encounter an `panic: invalid dimensions` error with ISO installation. See issue [#2937](https://github.com/harvester/harvester/issues/2937#issuecomment-1278545927) for a workaround.
```

File: docs/upload-image.md [hunk 11]
```diff
@@ -41,29 +41,28 @@ Currently, qcow2, raw, and ISO images are supported.
...
 
 ### Image StorageClass
 
-When creating an image, you can select a [StorageClass](./advanced/storageclass.md) to make the storage tiering of the image volume match the parameters of the StorageClass.
+When creating an image, you can select a [StorageClass](./advanced/storageclass.md) and use its pre-defined parameters like replicas, node selectors and disk selectors .
 
 :::note
 
-The image volume will not actually use the StorageClass selected here, as it's just a StorageClass parameters template.
+The image will not use the `StorageClass` selected here directly, it's just a `StorageClass` template.
```
Comment (vickyhella) [inline docs/upload-image.md:54]:
```suggestion
The image will not use the `StorageClass` selected here directly. It's just a `StorageClass` template.
```

File: docs/upload-image.md [hunk 12]
```diff
@@ -41,29 +41,28 @@ Currently, qcow2, raw, and ISO images are supported.
...
-In fact, the image volume will use a special StorageClass whose name has the  `longhorn-` prefix, which is automatically generated by the Harvester backend, but it will inherit the parameters from the StorageClass you select.
+Instead, it will create a special StorageClass under the hood with a prefix name of `longhorn-` , this is automatically done by the Harvester backend, but it will inherit the parameters from the StorageClass you have selected.
 
 :::
 
 ![](/img/v1.1/image-storageclass.png)
 
 ### Image Labels
 
+You can add labels to the image, which will help identify the OS type more accurately. Also, you can also add any custom labels for filtering if needed.
```
Comment (vickyhella) [inline docs/upload-image.md:64]:
```suggestion
You can add labels to the image, which will help identify the OS type more accurately. Also, you can add any custom labels for filtering if needed.
```

File: docs/upload-image.md [hunk 13]
```diff
@@ -41,29 +41,28 @@ Currently, qcow2, raw, and ISO images are supported.
...
 
-You can add labels to the image, which will help identify the OS type more accurately. Additionally, you can also add any custom labels when needed.
-
-If you create an image from a URL, the UI will automatically recognize the OS type and image category based on the image name. However, if you created the image by uploading a local file, you will need to manually select the corresponding labels.
+If your image name or URL contains any valid information, the UI will automatically recognize the OS type and image category for you. If not, you can also manually specify those corresponding labels on the UI.
```
Comment (vickyhella) [inline docs/upload-image.md:66]:
```suggestion
If your image name or URL contains valid information, the UI will automatically recognize the OS type and image category for you. If not, you can also manually specify those corresponding labels on the UI.
```

File: docs/advanced/pcidevices.md (hunk 4 above)
Comment (vickyhella) [reply docs/advanced/pcidevices.md:39]: `be careful not to use` seems a little confusing

**Review by vickyhella — APPROVED** [review]
Just a few nits. The rest LGTM!

File: docs/upload-image.md (hunk 12 above)
Comment (vickyhella) [reply docs/upload-image.md:64]: Duplicate `also`

File: docs/index.md [hunk 14]
```diff
@@ -75,22 +75,24 @@ During the installation, you can either choose to form a new cluster or join the
...
-8. Configure the login password of the host. The default SSH user is `rancher`.
-9. Recommended configuring the NTP server to make sure all nodes' times are synchronized. This defaults to `0.suse.pool.ntp.org`.
-10. (Optional) If you need to use an HTTP proxy to access the outside world, enter the proxy URL address here. Otherwise, leave this blank.
-11. (Optional) You can choose to import SSH keys from a remote server URL. Your GitHub public keys can be used with `https://github.com/<username>.keys`.
-12. (Optional) If you need to customize the host with a [Harvester configuration](./install/harvester-configuration.md) file, enter the HTTP URL here.
-13. Confirm the installation options and Harvester will be installed to your host. The installation may take a few minutes to complete.
-14. Once the installation is complete, the host will restart, and a console UI with management URL and status will be displayed. <small>(You can Use F12 to switch between the Harvester console and the Shell).</small>
-15. The default URL of the web interface is `https://your-virtual-ip`.
+5. Recommended choosing a separate disk for storing VM data.
```
Comment (vickyhella) [inline docs/index.md:78]:
```suggestion
5. We recommend choosing a separate disk for storing VM data.
```

File: docs/index.md [hunk 15]
```diff
@@ -75,22 +75,24 @@ During the installation, you can either choose to form a new cluster or join the
...
+
+   ![iso-installed.png](/img/v1.1/install/iso-nic-config.png)
+7. Optional: Configure the DNS servers; use commas as delimiters.
+8. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster <small>(Note: If your IP address is configured via DHCP, you will need to configure static MAC-to-IP address mapping on your DHCP server in order to have a persistent Virtual IP, VIP must be different than any Node IP)</small>.
+9. Configure the `cluster token`. This token will be used for adding other nodes to the cluster.
+10. Configure the login password of the host. The default SSH user is `rancher`.
+11. Recommended configuring the NTP server to make sure all nodes' times are synchronized. This defaults to `0.suse.pool.ntp.org`.
```
Comment (vickyhella) [inline docs/index.md:87]:
```suggestion
11. We recommend configuring the NTP server to make sure all nodes' times are synchronized. This defaults to `0.suse.pool.ntp.org`.
```

File: docs/index.md [hunk 16]
```diff
@@ -50,7 +50,7 @@ To get the Harvester server up and running, the following minimum hardware is re
...
 |:---|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| CPU | x86_64 only. Hardware-assisted virtualization is required. 8-core processor minimum; 16-core or above preferred                                                                                            |
+| CPU | x86_64 only. Hardware-assisted virtualization is required. 8-core processor minimum for testing; 16-core or above preferred for production                                                                |
 | Memory | 32 GB minimum; 64 GB or above preferred                                                                                                                                                                    |
 | Disk Capacity | 140 GB minimum for testing; 500 GB or above preferred for production                                                                                                                                       |
```
Comment (bk201) [inline docs/index.md:55]: 200GB for testing upgrade.

File: docs/rancher/rancher-integration.md [hunk 17]
```diff
@@ -71,7 +72,7 @@ To install Rancher with Docker:
 
 ```shell
 # for a quick evaluation, you can run the Rancher server with the following command
-$ sudo docker run -d --restart=unless-stopped -p 80:80 -p 443:443 --privileged rancher/rancher:v2.6.6
+$ sudo docker run -d --restart=unless-stopped -v /opt/rancher:/var/lib/ranche -p 80:80 -p 443:443 --privileged rancher/rancher:v2.6.9
```
Comment (bk201) [inline docs/rancher/rancher-integration.md:75]:
```suggestion
$ sudo docker run -d --restart=unless-stopped -v /opt/rancher:/var/lib/rancher -p 80:80 -p 443:443 --privileged rancher/rancher:v2.6.9
```

Not entirely sure.

File: docs/rancher/virtualization-management.md [hunk 18]
```diff
@@ -36,7 +36,7 @@ For Harvester v0.3.0 and above, virtualization management with the multi-cluster
 
     ```shell
     # for a quick evaluation, you can run the Rancher server with the following command
-    $ sudo docker run -d --restart=unless-stopped -p 80:80 -p 443:443 --privileged rancher/rancher:v2.6.6
+    $ sudo docker run -d --restart=unless-stopped -v /opt/rancher:/var/lib/ranche -p 80:80 -p 443:443 --privileged rancher/rancher:v2.6.9
```
Comment (bk201) [inline docs/rancher/virtualization-management.md:39]: ditto

Comment (bk201) [inline docs/rancher/virtualization-management.md:39]:
```suggestion
    $ sudo docker run -d --restart=unless-stopped -v /opt/rancher:/var/lib/rancher -p 80:80 -p 443:443 --privileged rancher/rancher:v2.6.9
```

File: docs/index.md [hunk 19]
```diff
@@ -75,22 +75,24 @@ During the installation, you can either choose to form a new cluster or join the
...
-15. The default URL of the web interface is `https://your-virtual-ip`.
+5. Recommended choosing a separate disk for storing VM data.
+   ![iso-install-disk.png](/img/v1.1/install/iso-select-data-disk.png )
+6. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster, in case the DHCP is used, the user must make sure the DHCP server always offers the same IP for the same Node. Due to a changed Node IP the related Node can not join the cluster, or even break the cluster)</small>.
+
+   ![iso-installed.png](/img/v1.1/install/iso-nic-config.png)
+7. Optional: Configure the DNS servers; use commas as delimiters.
+8. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster <small>(Note: If your IP address is configured via DHCP, you will need to configure static MAC-to-IP address mapping on your DHCP server in order to have a persistent Virtual IP, VIP must be different than any Node IP)</small>.
```
[the remark this answers is not in the document]
Comment (guangbochen) [reply docs/index.md:84]: This changes the original meaning, so I'll skip this one.

File: docs/index.md (hunk 16 above)
Comment (guangbochen) [reply docs/index.md:55]: Sure will update the minimum to 200GB

**Comment by guangbochen** [comment]
Updated and squaished, thanks

**Review by bk201 — APPROVED** [review]
LGTM!

**Pull request merged**